packages/e2e-tests/config/setup-screenshots.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const root = process.env.GITHUB_WORKSPACE || '../../../';
+const SCREENSHOTS_PATH = root + '/screenshots';
+
+if ( ! fs.existsSync( SCREENSHOTS_PATH ) ) {
+	fs.mkdirSync( SCREENSHOTS_PATH );
+}
+
+/**
+ * @copyright Tom Esterez (@testerez) https://github.com/smooth-code/jest-puppeteer/issues/131#issuecomment-424073620
```

Comment:
oh, also any concerns about copyright or licensing here?

Reply:
I ended up refactoring things quite a bit so probably not, however I still left an acknowledgement note.

Reply:
CC @testerez - your idea from https://github.com/smooth-code/jest-puppeteer/issues/131#issuecomment-424073620 is pretty awesome, we would like to use it here! :-)

Reply:
Appreciate it, thanks!